tests/test_extractor_tfds.py

```diff
@@ -98,6 +100,38 @@ def test_can_extract_cifar10(self):
     def test_can_extract_cifar100(self):
         self._test_can_extract_cifar('cifar100')
 
+    @mark_requirement(Requirements.DATUM_GENERAL_REQ)
+    def test_can_extract_imagenet_v2(self):
+        with mock_tfds_data():
+            tfds_ds, tfds_info = tfds.load(
+                'imagenet_v2', split='train', with_info=True,
+                # We can't let TFDS decode the image for us, because:
+                # a) imagenet_v2 produces JPEG-encoded images;
+                # b) TFDS decodes them via TensorFlow;
+                # c) Datumaro decodes them via OpenCV.
+                # So for the decoded results to match, we have to decode
+                # them via OpenCV as well.
```

Comment:
Why decoding results can be different?

Reply:
I don't know. Different implementations, I guess.

Reply:
I looked into it, just in case. The implementations are the same (libjpeg), but TensorFlow selects a faster/less accurate decoding method: https://github.com/tensorflow/tensorflow/blob/v2.7.0/tensorflow/core/kernels/image/decode_image_op.cc#L117. As far as I can see, OpenCV uses the default method.

Reply:
Ok, thanks